offline/offline_classification.py

```diff
@@ -0,0 +1,61 @@
+# %%
+'''
+Offline classification. Tested classifiers so far: k-fold LDA
+
+Author:
+    Ayca Kepce
+    Nathan van Beelen
+
+'''
+
+# !%matplotlib qt
+
+import numpy as np
+
+import pickle
+
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
+
+from utils import process_features, epoching, k_fold_LDA
+
+# %%
+# Alternatively load the concatenated data
+file = r'C:\Users\AYCA\PycharmProjects\biosignal_stars\data\sub_epochs\sub_epochs_trial.pkl'
```

Comment:
Can you change all the absolute paths to relative ones? If directories have to be traversed it might be best to implement an OS agnostic method (see https://stackoverflow.com/a/6036156 for example).

Reply:
The data now is under the directory "data\sub_epochs", thus it is the same for all users.

Reply:
Thank you! It's not OS agnostic yet however. The difference is that the slash is used in Linux between directory names, but a backward slash is used in Windows. So on my Linux machine Python still cannot find the file.